assets/sass/components/surveys/_googlesitekit-surveys.scss

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Perhaps we need to rename this file to `_googlesitekit-survey-modal.scss` to be more precise?

Reply:
Yeah, can be. I renamed the file. I was a bit reluctant to do so because there are only a few lines of CSS for the modal but that's fine I guess.